README.md

```diff
@@ -0,0 +1,2 @@
+# *imhungry* Android app
```

Comment:
I don't believe we should say Android app in the headline directly, but could mention it in the body.
Perhaps something like "*imhungry* - Your Mealtime Companion," or something more directly related to the theme of the app.

See: https://github.com/UweTrottmann/SeriesGuide

If we edit the tagline of the repo to mention the platform, then I don't believe we should directly have it in the header of the readme.